src/ibex_bluesky_core/devices/dae/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""Utilities for the DAE device - mostly XML helpers."""
+
+from __future__ import annotations
```

Comment:
Did we actually need this?

https://peps.python.org/pep-0649/ is accepted and deprecates https://peps.python.org/pep-0563/ ...

Reply:
don't think so, IDE probably added it